src/utils/_opacity.scss

```diff
@@ -4,9 +4,31 @@
 // @include bgcolor-opacity($c-black);
 
 @mixin bgcolor-opacity ($color) {
+
+  background-color: rgba($color, 1);
+
   @each $value in $c-opacity {
     &-#{$value} {
       background-color: rgba($color, $value/100);
     }
   }
 }
+
+@mixin bgcolor-overlay-opacity ($color) {
```

Comment:
What do we need it for ?
Can you show when you use it?

Reply:
the mixin will be used in the Image Panel